2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -0,0 +1,203 @@
+#import "template.typ": *
+
+#show: project.with(
+  title:"Piano di progetto",
+  subTitle: "Pianificazione, analisi e retrospettive", 
+  docType: "doc",
+  partecipants: (
+  (name: "Banzato Alessio", role: "Verificatore"),
+  (name: "Carraro Riccardo", role: "Redattore"),
+  (name: "Gardin Giovanni",  role: "Redattore"),
+  (name: "Nardo Silvio",   role: "Verificatore"),
+  (name: "Oseliero Antonio",   role: "Responsabile"),
+  (name: "Todesco Mattia",  role: "Verificatore"),
+  (name: "Zaccone Rosario",  role: "Amministratore"),
+  ),
+  showIndex: true,
+);
+#set text(font:"New Computer Modern")
+#set align(left)
+#show "link": word => text[#text(fill:blue, word)]
+
+= Introduzione
+Il documento _Piano di progetto_ ha il delicato compito di governare la pianificazione dell'avanzamento del progetto, determinando task e obiettivi da raggiungere, presentando un'analisi critica del lavoro fino a quel momento svolto. L'intento è rendicontare l'intero sviluppo, valutando criticamente l'operato compiuto per eventualmente miglirarlo, e gestire in modo efficace ed efficiente le risorse.
+
+Il documento si articola in 5 sezioni, quali:
+- *Rischi e loro mitigazione*: dedicata all'analisi dei rischi che possono insorgere con relativi sistemi di mitigazioni;
+- *Divisione temporale di sviluppo*: dedicata all'analisi della suddivisione temporale dello sviluppo del progetto;
```

Comment:
Cercherei di spiegare in modo più approfondito cosa c'è in questa sezione
```suggestion
- *Divisione temporale di sviluppo*: dedicata all'analisi della suddivisione temporale dello sviluppo del progetto;
```

Reply:
E' presente la sezione dedicata